pyproject.toml

```diff
@@ -1,3 +1,58 @@
+[tool.poetry]
+name = "pytest-split"
+version = "0.2.1"
```

Comment:
How is this updated? Manually?

Reply:
Yeah so when I update a package I would manually indent the version here and publish a release in the github repo 🙂

Reply:
I wonder if there's some way to get the version from git tags with poetry 🤔  But yeah, I can look into that later.

Reply:
https://github.com/tiangolo/poetry-version-plugin

Reply:
If you want to add this in a follow-up PR you'd just need to change the test and publish action to this as well, since it requires >= 1.2.0a1

```yaml
      - name: Install poetry
        uses: snok/install-poetry@v1.1.6
        with:
          version: 1.2.0a1
          virtualenvs-in-project: true

```